cmd/object-api-utils.go

```diff
@@ -296,6 +299,66 @@ func getRandomHostPort(records []dns.SrvRecord) (string, int) {
 	return srvRecord.Host, srvRecord.Port
 }
 
+// IsCompressed returns true if the object is marked as compressed.
+func (o *ObjectInfo) IsCompressed() bool {
+	_, ok := o.UserDefined[ReservedMetadataPrefix+"compression"]
+	return ok
+}
+
+// GetActualSize - read the decompressed size from the meta json.
+func (o *ObjectInfo) GetActualSize() int64 {
+	metadata := o.UserDefined
+	sizeStr, ok := metadata[ReservedMetadataPrefix+"actualSize"]
```

Comment:
Can we use a better naming scheme here? Like:
`X-Minio-Internal-Compression-Actual-Size` ? And prefix everything related to compression with `Compression`?
Otherwise it may become very messy...

Reply:
I agree

Reply:
@aead  @harshavardhana  Since `ActualSize` is no longer compression specific now . Should we actually change this to `X-Minio-Internal-Compression-*` ?